### PR TITLE
fix(parser): fix version()

### PR DESCRIPTION
**Pull request by sandflee**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary
`select version_xx from table`  generates databend version info.

### CODE REVIEW

File: src/query/service/src/servers/mysql/mysql_federated.rs [hunk 1]
```diff
@@ -177,7 +177,7 @@ impl MySQLFederated {
     fn federated_mixed_check(&self, query: &str) -> Option<DataBlock> {
         let rules: Vec<(&str, Option<DataBlock>)> = vec![
             (
-                "(?i)^(SELECT VERSION())",
+                r"(?i)^(SELECT VERSION\(\s*\))",
```
Comment (BohuTANG) [inline src/query/service/src/servers/mysql/mysql_federated.rs:180]: Can you give a raw query sent from MySQL?

Comment (sandflee) [reply src/query/service/src/servers/mysql/mysql_federated.rs:180]:
```
SELECT version_name,\n       os,\n       p_date, file_type, video_succ_cnt\nFROM xxx\nWHERE p_date >= '20190101'\n  AND p_date <= '20191010'\nORDER BY version_name,\n         os,\n         p_date, file_type, video_succ_cnt\nLIMIT 100
```

Comment (Xuanwo) [reply src/query/service/src/servers/mysql/mysql_federated.rs:180]: The old regex `(?i)^(SELECT VERSION())` is wrong and will match all query that start with `select version`

image

Comment (BohuTANG) [reply src/query/service/src/servers/mysql/mysql_federated.rs:180]: 👍 
Could you add a test for this query in https://github.com/datafuselabs/databend/blob/main/src/query/service/tests/it/servers/mysql/mysql_federated.rs

Comment (sandflee) [reply src/query/service/src/servers/mysql/mysql_federated.rs:180]: updated

**Review by Xuanwo — APPROVED** [review]

**Review by b41sh — APPROVED** [review]

**Pull request merged**